exercises/practice/leap/.approaches/cond-form/snippet.txt

```diff
@@ -0,0 +1,6 @@
+(define (leap-year? year)
+  (cond
+    [(zero? (modulo year 400)) #t]
+    [(not (zero? (modulo year 100))) #f]
```

Comment:
This clause should not be negated. Should just be `[(zero? (modulo year 100)) #f]`.

Reply:
Thanks for the catch. It was a copy and paste error from the boolean chain approach.